src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -2532,9 +2904,47 @@ private static T DeserializeString<T>(string stringToDeserialize, bool shouldRep
         }
     }
 
+    private static T VerifyIndentationOfSerializedXml<T>(T value, string baseline, string indentChars = null, DataContractJsonSerializerSettings settings = null, Func<DataContractJsonSerializer> serializerFactory = null, bool skipStringCompare = false)
```

Comment:
How's this method different from SerializeAndDeserialize? Can we use SerializeAndDeserialize instead?